src/test/resources/features/reference_domains.feature

```diff
@@ -37,14 +37,14 @@ Feature: Reference Domains
 
   Scenario: Retrieve used case note types with sub-types
     When request submitted to retrieve used case note types
-    Then "31" reference code items are returned
-    And domain for all returned items is "TASK_TYPE"
-    And description for "1st" returned item is "Accredited Programme"
```

Comment:
If we do care about the ordering of the results then using shared data is not the way to test this, so I've removed all positional indexes from this test.